src/cmd-buildupload

```diff
@@ -51,7 +51,13 @@ def parse_args():
                     "and set Content-Disposition names", action='store_true')
     s3.set_defaults(func=cmd_upload_s3)
 
-    return parser.parse_args()
+    args = parser.parse_args()
+    if args.dry_run:
```

Comment:
We should then be able to drop this diff hunk too.

Reply:
:+1: pushed.